libraries/CurieSoftwareSerial/SoftwareSerial.cpp

```diff
@@ -94,14 +94,21 @@ bool SoftwareSerial::listen()
     bufferOverflow = false;
     _receive_buffer_head = _receive_buffer_tail = 0;
     active_object = this;
-    rxPin = _receivePin;
+    _rxPin = _receivePin;
+    rxIntraBitDelay = _rx_delay_intrabit;
+    rxCenteringDelay = _rx_delay_centering;
```

Comment:
The reason I used globals is because recv() which acts as the interrupt handler is static.
Therefore, it cannot access those variables if I kept them as member variables of the class.

Do you think having those variables names "reserved" can be an issue? If so I can always rename them to something else.

Also if you have a better solution, Pull Requests are always welcome. :)